extendr-api/src/wrapper/altrep.rs

```diff
@@ -155,7 +155,7 @@ pub trait AltrepImpl: Clone + std::fmt::Debug {
 fn manifest(x: SEXP) -> SEXP {
     single_threaded(|| unsafe {
         Rf_protect(x);
-        let len = XLENGTH_EX(x);
```

Comment:
This appears which i think is an a-ok location? 
https://github.com/r-devel/r-svn/blob/48e02881502dce9eeeefff1bc422bc0968913c04/src/include/Rinternals.h#L1057C34-L1057C34

Reply:
The link doesn't provide exact location. But yes?